apps/allocations/app/components/App/App.js

```diff
@@ -110,59 +111,76 @@ class App extends React.Component {
     this.setState({ panel: { visible: false } })
   }
 
+  handleResolveLocalIdentity = address => {
+    return this.props.api.resolveAddressIdentity(address).toPromise()
+  }
+
+  handleShowLocalIdentityModal = address => {
+    return this.props.api
+      .requestAddressIdentityModification(address)
+      .toPromise()
+  }
+
   render() {
     const { panel } = this.state
     const { displayMenuButton = false } = this.props
     const PanelContent = panel.content
+
+    console.log(this.props)
```

Comment:
Should we clean console.logs? Or at least put them into an intermediate environment conditional "log" function as John and you brilliantly did on the profiles app

Reply:
Cleaned. A agree conditional debugging will be a great thing to add - although... prior to introducing it we need to discuss strategy and do it properly :)